src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -2295,7 +2295,7 @@ public void LongProcessNamesAreSupported()
                 Process[] runningProcesses = Process.GetProcesses();
                 try
                 {
-                    Assert.Contains(runningProcesses, p => p.ProcessName == LongProcessName);
+                    Assert.Contains(runningProcesses, p => p.ProcessName.Contains(LongProcessName));
```

Comment:
Why was this change necessary?

Reply:
I did some debugging on that test and it fails because the test checks for an exact match of the process name string "123456789012345678901234567890" but the actual process name has the full path, eg.
```
    /tmp/#ProcessTests_msso5zem.fzn/123456789012345678901234567890
```
so I guessed this should match the substring.

Is os-specific code supposed to return the process name without the leading path?
Thanks

Reply:
Yes it should be just the process name. https://github.com/dotnet/runtime/pull/105403/files#r1690385726 should fix it.